cli/integration-test/integration/lsp.clj

```diff
@@ -153,3 +153,19 @@
         (do
           (Thread/sleep 500)
           (recur))))))
+
+(defn await-request [method]
```

Comment:
Maybe this to let it more clear?
```suggestion
(defn await-client-request [method]
```

Reply:
Agreed, these names are really confusing. I find it hard to keep track of who initiates communications. Anyway, I like your proposed name better. I'll update that.